### ISSUE TITLE
Function attributes - Support for multiline syntax 

### ISSUE THREAD

**Issue by Lunardis** [Makopo/sublime-text-lsl#37]
This is how I write functions in LSL, works just fine with your plugin:

```
function myFunction (string doing, string thing, string character) {
    llOwnerSay("This is " + doing + " " + thing + character );
}
```

But this is how a friend of mine wrotes functions:

```
function myFunction (
    string doing,
    string thing,
    string character
) {
    llOwnerSay("This is " + doing + " " + thing + character );
}
```

It seems LSL don't have a problem with this and I can understand why she need this.
But your plugin (sublimes goto-feature don't work well with such attributes in function) seems not to understand this kind of multiline-attribute syntax.

So I thought to report this issue.

**Comment by Makopo** [comment]
Hi, Lunardis. Thank you for reporting.
Can you try current version 2.3.1 to confirm if the issue is solved and no other issues happening?

**Comment by Lunardis** [comment]
So it was one character to resolve. *sighs* We tried so complex regex-patterns, and you fixed this with just the deletion of one character. *smirks*

Thank you! We tried it and it worked.

**Comment by buildersbrewery** [comment]
Reference links:

* Partly related to #19, which was fixed in [v2.3.2](https://github.com/Makopo/sublime-text-lsl/releases/tag/2.3.2).
